contracts/LeftGalleryController.sol

```diff
@@ -126,6 +130,7 @@ contract LeftGalleryController is Ownable {
             works[workId].artist,
             works[workId].editions,
             works[workId].price,
+            works[workId].adminSplit,
```

Comment:
Would it make the contract 'cheaper' if we define an internal private method that emits the `updatedWork` event? And we call that instead each time the work is updated? I see some repetition.

Reply:
Would you mind elaborate on the internal private method?